### PR TITLE
Refactor create_cluster tests to share expected values

### PR DESCRIPTION
**Pull request by dalees**
The Helm driver under test has a large number of common values, each test should only need to define those it cares about. This makes refactoring and adding new features much easier and reduces duplication.

### CODE REVIEW

**Comment by dalees** [comment]
@mkjpryor for review - I'll update my other PRs if this merges before!

**Review by mkjpryor — APPROVED** [review]
LGTM. Reducing unnecessary repetition is always good.

File: magnum_capi_helm/tests/test_driver.py [hunk 1]
```diff
@@ -1094,6 +1094,52 @@ def test_get_chart_version_from_template(self):
 
         self.assertEqual("1.42.0", version)
 
+    def get_cluster_helm_standard_values(self):
```
Comment (JohnGarbutt) [inline magnum_capi_helm/tests/test_driver.py:1097]: Nit: I would prefer _get_cluster_helm_standard_values(), but otherwise this seems good to me.

**Review by JohnGarbutt — APPROVED** [review]

File: magnum_capi_helm/tests/test_driver.py (hunk 1 above)
Comment (dalees) [reply magnum_capi_helm/tests/test_driver.py:1097]: Good point, thanks; done as part of https://github.com/stackhpc/magnum-capi-helm/pull/23

**Pull request merged**